enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -214,4 +259,24 @@ public RoleResult( String role, Set<String> users )
             this.users.addAll( users );
         }
     }
+
+    public class TransactionResult
+    {
+        public final String username;
+        public final String transaction;
+
+        public TransactionResult( KernelTransaction tx )
+        {
+            if ( tx.mode() instanceof AuthSubject )
+            {
+                AuthSubject authSubject = (AuthSubject) tx.mode();
+                username = authSubject.name();
+            }
+            else
+            {
+                username = "-";
+            }
+            transaction = tx.toString();
```

Comment:
I do not think it is valuable for user to see things like `KernelTransaction[1]`. Users should actually never see `KernelTrasnaction` because it is a name of internal API. Can we instead return a count of transactions for the given user and a count of killed transactions?